src/Components/SavedSearchAlert/Components/AddFiltersRarity.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import { Box, Checkbox, Flex, Spacer, Text } from "@artsy/palette"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+
+export const RARITY_OPTIONS = [
```

Comment:
Same, could we use the options from [RarityOptions](https://github.com/artsy/force/blob/20165eac3335ff5e4a352d63cc4f7d394a51e210/src/Apps/MyCollection/Routes/EditArtwork/Utils/rarityOptions.ts#L1)?